src/installer/pkg/projects/netcoreapp/src/localnetcoreapp.override.targets

```diff
@@ -93,8 +93,8 @@
 
       <CoreFXReferenceCopyLocalItems
         Include="
-          $([MSBuild]::NormalizeDirectory('$(CoreFXOverridePath)', 'bin', 'pkg', 'netcoreapp', 'runtime', '$(LibrariesOSGroup)-$(Configuration)-$(TargetArchitecture)'))*;
-          $([MSBuild]::NormalizeDirectory('$(CoreFXOverridePath)', 'bin', 'native', 'netcoreapp-$(LibrariesOSGroup)-$(Configuration)-$(TargetArchitecture)'))*"
+          $([MSBuild]::NormalizeDirectory('$(CoreFXOverridePath)', 'bin', 'pkg', 'netcoreapp5.0', 'runtime', '$(LibrariesOSGroup)-$(Configuration)-$(TargetArchitecture)'))*;
```

Comment:
Why not use $(NetCoreAppCurrent) here? Is it not defined here?

Reply:
Its defined, I thought it wasn't. I can change that in a follow up PR (Just want to get this in before more merge conflicts)

Reply:
There is still more work required here, I would make the change as part of this PR.